doc/user/compatibility.md

```diff
@@ -16,6 +16,14 @@ if we could design the language better. If we can support a feature, we will do.
 In the future we aim to provide compatibility with extra functionality provided
 by JRuby, but at the moment we do not.
 
+## Identification
+
+TruffleRuby identifies as a `RUBY_ENGINE` and related constants of
+`truffleruby`. The `RUBY_VERSION` and related constants are as our compatible
+MRI version. `RUBY_REVISION` and `RUBY_PATCHLEVEL` are always zero.
+`RUBY_RELEASE_DATE` is the current year. `ENGINE_VERSION` is the GraalVM
+version, or `0.0` if your build is not part of a GraalVM release.
```

Comment:
I think this is fairly hard to read.
Maybe it should just be like:

TruffleRuby defines RUBY_* constants for identification as such:
```ruby
RUBY_ENGINE = "truffleruby"
RUBY_VERSION = "X.Y.Z" # compatible MRI version
RUBY_REVISION = 0
RUBY_PATCHLEVEL = 0
...
```

Reply:
Changed it to a list.